nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -110,6 +110,30 @@ def get_bits_length(config, quant_type):
     else:
         return config["quant_bits"].get(quant_type)
 
+def del_simulated_attr(module):
+    """
+    delete redundant parameters in quantize module
+    """
+    if hasattr(module, 'old_weight'):
+        delattr(module, 'old_weight')
+    if hasattr(module, 'ema_decay'):
+        delattr(module, 'ema_decay')
+    if hasattr(module, 'tracked_min_biased'):
+        delattr(module, 'tracked_min_biased')
+    if hasattr(module, 'tracked_max_biased'):
+        delattr(module, 'tracked_max_biased')
+    if hasattr(module, 'tracked_min'):
+        delattr(module, 'tracked_min')
+    if hasattr(module, 'tracked_max'):
+        delattr(module, 'tracked_max')
+    if hasattr(module, 'scale'):
+        delattr(module, 'scale')
+    if hasattr(module, 'zero_point'):
+        delattr(module, 'zero_point')
+    if hasattr(module, 'weight_bit'):
+        delattr(module, 'weight_bit')
+    if hasattr(module, 'activation_bit'):
+        delattr(module, 'activation_bit')
```

Comment:
how do you choose these attributes? different quantization algorithms have different subset of these attributes? is it possible that a new quantization algorithm has more attributes?

Reply:
This function is only for QAT now, other quantization algorithms might have other attributes and we need to define another function to handle it.

Reply:
then it is better to put it in `QAT_Quantizer`

Reply:
BTW, why only support export for QAT?

Reply:
Because QAT is enough for **linear quantization** simulation. **Binarized quantization** like BNN needs no calibration and users can save it weights as normal.

Reply:
could you refer to model export implemented in pruners, and try to align the export feature of pruner and quantizer?

Reply:
Have aligned `export_model()` function in pruners and quantizers.